internal/pkg/armeria/command-handlers.go

```diff
@@ -134,6 +134,76 @@ func handleLookCommand(ctx *CommandContext) {
 	}
 }
 
+func handleGlanceCommand(ctx *CommandContext) {
+	r := ctx.Character.Room()
+
+	var objNames []string
+	for _, o := range r.Here().All() {
+		obj := o.(ContainerObject)
+
+		if obj.Type() == ContainerObjectTypeCharacter && obj.(*Character).Player() == nil {
```

Comment:
I recently added a helper function for this so instead of checking if player is nil to determine if they are online or not, ie: 
```
obj.(*Character).Player() == nil
```
You can now just do:
```
!obj.(*Character).Online()
```
You don't have to change this in this PR though. I'll likely go and update this across-the-board at some point.